src/pytest_cov/engine.py

```diff
@@ -151,10 +150,20 @@ def start(self):
         self.cov = coverage.Coverage(source=self.cov_source,
                                      branch=self.cov_branch,
                                      config_file=self.cov_config)
+
+        # Get absolute location of used config file.
+        parsed_config_file = self.cov.config.config_file
+        if parsed_config_file:
+            parsed_config_file = os.path.abspath(parsed_config_file)
+            assert os.path.exists(parsed_config_file)
```

Comment:
I think this would cause failures when coverage config is completely missing. I suspect this is hosing up all the tests.

Reply:
Tests are mostly broken because it requires coveragepy 5.